pkg/storage/emptydir/emptydir.go

```diff
@@ -51,9 +53,25 @@ func (d *driver) Volumes() ([]corev1.Volume, []corev1.VolumeMount, error) {
 	return []corev1.Volume{vol}, []corev1.VolumeMount{mount}, nil
 }
 
-func (d *driver) CompleteConfiguration(customResourceStatus *opapi.ImageRegistryStatus) error {
+func (d *driver) StorageExists(cr *opapi.ImageRegistry) error {
+	return nil
+}
+
+func (d *driver) CreateStorage(cr *opapi.ImageRegistry) error {
+	return nil
+}
+
+func (d *driver) RemoveStorage(cr *opapi.ImageRegistry) error {
+	if !cr.Status.Storage.Managed {
+		return fmt.Errorf("storage is not managed by the image registry operator, so we can't delete it.")
```

Comment:
are we currently setting Managed=true for emptydir storage?  Basically we should either always set it to true, or this should not care whether it is managed or not (since removing storage for emptydir is a no-op anyway)